code-challenges.test.js

```diff
@@ -71,3 +129,26 @@ const accountTransactions3 = []
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+
+// 1.) Create a function that takes in an array (parameter) and takes in any array 
+
+const accumSum = (array) => {
+// 2.) Create a for loop where the sum, that starts at 0, is added to the elements and returns the new sum into the empty array. The for loop will iterate over the length of the array. 
+  let sum = 0 
+  let newSum = []
+  for (let i = 0; i<array.length; i++) {
+    
+// 3.) Create the logic in the for loop by having the sum, 0, plus the value of the element to equal a sum. Then to take that value and push it to the value in the next index and add those values together.
```

Comment:
Confusing pseudo code here. An example to make it clear would be:
- update sum per iteration to equal current sum plus value at current index
- push the updated sum into newSum array